frontend/src/components/GddSend/TransactionForm.vue

```diff
@@ -147,24 +153,26 @@ export default {
     InputIdentifier,
     InputAmount,
     InputTextarea,
+    CommunitySwitch,
   },
   props: {
     balance: { type: Number, default: 0 },
     identifier: { type: String, default: '' },
     amount: { type: Number, default: 0 },
     memo: { type: String, default: '' },
     selected: { type: String, default: 'send' },
+    targetCommunity: { type: Object, default: { id: 0, name: COMMUNITY_NAME } },
```

Comment:
is this correct to set the default selected community from configuration than from the previous read community-list?

Reply:
The CommunitySwitch component reads the community list and is the only one that can access it unless I pass it on. But I see no reason for that. Outside the select component I only need the choosen community.